framework/interface/templates/home.html

```diff
@@ -3,21 +3,7 @@
 {% block title %}Home Page{% end %}
 
 {% block content %}
-<div class="container-fluid jumbotron">
```

Comment:
@delta24  In this PR I have removed the jumbo-tron because by the end of this weak I will complete the home page after that we will not need jumbotron.
I have tried to put the pie chart below the jumbo tron but it was not looking good. IMO we should jumbo tron because it has no use.